record/record.controller.js

```diff
@@ -33,6 +33,14 @@
             }
         };
 
+        vm.copyRecord = function() {
+            var referenceFilter = $rootScope.reference.location.filter;
```

Comment:
Is `referenceFilter` being used anywhere else?/What is `referenceFilter` being used for?